### PR TITLE
chore: bump glog to 0.7.1

### PR DESCRIPTION
**Pull request by aleksraiden**
Update google log to 0.7.0. Full release notes here: https://github.com/google/glog/releases/tag/v0.7.0 
Also, included update 0.7.1 (see here: https://github.com/google/glog/releases/tag/v0.7.1)

**Key features**

- C++14 is now the primary language standard
- google::EnableLogCleaner(unsigned) has been superseded by google::EnableLogCleaner(std::chrono::minutes)
- google::LogSeverity has been converted to an unscoped enum. This can result in compilation issues if implicit conversion between an int and google::LogSeverity was erroneously assumed.
- A lot of fixes and enhancements (see full changelog)

**Code changes**

At this update we need to fix our codebase to have full comparability with this and futures release of glog. Thanks @torwig for this fixes.

- Fix google::EnableLogCleaner (done)
- Fix linting and code checks (done)
- Fix google::LogSeverity (done)
- Fix a symbolizes a program counter (in progress)

### CODE REVIEW

**Comment by aleksraiden** [comment]
Now we have unresolved issue at build kvrocks:

> [100%] Linking CXX executable kvrocks
> /usr/sbin/ld: CMakeFiles/kvrocks.dir/src/cli/main.cc.o: in function `SegvHandler':
> /__w/kvrocks/kvrocks/src/cli/signal_util.h:54:(.text.SegvHandler[SegvHandler]+0x266): undefined reference to `google::Symbolize(void*, char*, unsigned long)'
> collect2: error: ld returned 1 exit status

**Comment by torwig** [comment]
`google::Symbolize` isn't exported anymore in 0.7.0 so we need to use something else for our purposes (or ask the maintainers of GLOG to export it again :) ).

File: src/cli/main.cc [hunk 1]
```diff
@@ -103,15 +104,17 @@ static void InitGoogleLog(const Config *config) {
...
     FLAGS_stderrthreshold = google::ERROR;
     FLAGS_logtostdout = true;
     std::setbuf(stdout, nullptr);
   } else {
     FLAGS_log_dir = config->log_dir + "/";
     if (config->log_retention_days != -1) {
-      google::EnableLogCleaner(config->log_retention_days);
+      auto minutes =
+          std::chrono::duration_cast<std::chrono::minutes>(std::chrono::hours(24) * config->log_retention_days);
+      google::EnableLogCleaner(minutes);
```
Comment (PragmaTwice) [inline src/cli/main.cc]:
```suggestion
      google::EnableLogCleaner(std::chrono::hours(24) * config->log_retention_days);
```

File: src/config/config.cc [hunk 2]
```diff
@@ -553,7 +553,9 @@ void Config::initFieldCallback() {
              }
 
              if (log_retention_days != -1) {
-               google::EnableLogCleaner(log_retention_days);
+               auto minutes =
+                   std::chrono::duration_cast<std::chrono::minutes>(std::chrono::hours(24) * log_retention_days);
+               google::EnableLogCleaner(minutes);
```
Comment (PragmaTwice) [inline src/config/config.cc]:
```suggestion
               google::EnableLogCleaner(std::chrono::hours(24) * log_retention_days);
```

**Comment by aleksraiden** [comment]
Hmm, I think (thanks, @torwig) that in failed gotest kvrocks not starting yet. Need to investigate this

**Comment by aleksraiden** [comment]
Great thanks to @torwig about rewrite a gotest for logger. Wuuf, now a longest PR ready to review

**Comment by git-hulk** [comment]
@aleksraiden Great!

**Review by git-hulk — APPROVED** [review]

**Review by torwig — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Pull request merged**